tests/create_test_data.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, Tuple
+
+import numpy as np
+import xarray as xr
+from numpy.random import default_rng
+
+
+def create_data(
+    shape: Tuple[int, ...],
+    chunks: Dict[str, int] = {},
+    skin_correction: bool = False,
+    order: str = "F",
+    use_xr=True,
+    land_mask=False,
+):
+    size = shape[0] * shape[1]
+    shape2d = (shape[0], shape[1])
+    rng = default_rng()
+    indices = rng.choice(size, size=int(size * 0.3), replace=False)
+    multi_indices = np.unravel_index(indices, shape2d)
+
+    def _arr(value, chunks, order):
+        arr = np.full(shape, value, order=order)
+        if land_mask:
+            arr[
+                multi_indices[0], multi_indices[1], :
+            ] = np.nan  # add NaNs to mimic land mask
+        if use_xr:
+            arr = xr.DataArray(arr)
+            if chunks:
+                arr = arr.chunk(chunks)
+
+        # adds random noise scaled by a percentage of the value
+        randomize_factor = 0.001
+        randomize_range = value * randomize_factor
+        noise = np.random.rand(*shape) * randomize_range
+        arr = arr + noise
```

Comment:
```suggestion
        # adds random noise scaled by a percentage of the value
        randomize_factor = 0.001
        randomize_range = value * randomize_factor
        noise = np.random.rand(*shape) * randomize_range
        arr = arr + noise
        
        if land_mask:
            arr[
                multi_indices[0], multi_indices[1], :
            ] = np.nan  # add NaNs to mimic land mask
        if use_xr:
            arr = xr.DataArray(arr)
            if chunks:
                arr = arr.chunk(chunks)
```

Reply:
I guess this was technically fine before, but I think we should do the randomization before the xarray wrapping/chunking. Just to avoid potential trouble with dask along the way.